test/test_decomp.py

```diff
@@ -279,6 +279,8 @@ def normalize_op_input_output(f, sample, requires_grad=True):
     # CompositeAutogradImplicit
     # See https://github.com/pytorch/pytorch/issues/81669
     (None, None, "nn.functional.relu6"),
+    (None, None, "nn.functional.mish"),
+    (None, None, "nn.functional.silu"),
```

Comment:
just curious - what caused these to change?

Reply:
They're now getting decomposed, whereas previously we were incorrectly not decomposing them